CMakeLists.txt

```diff
@@ -174,3 +180,8 @@ endif()
 if (${VKB_BUILD_VULKAN_VALIDATION_LAYERS})
   include(cmake/vulkan-validation-layers.cmake)
 endif()
+
+# Vulkan-ExtensionLayer requires Vulkan-Headers
```

Comment:
I don't think this is necessary, most of the things in the Vulkan section require headers to come first.